satellite/payments/stripecoinpayments/service.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package stripecoinpayments
+
+import (
+	"context"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/stripe/stripe-go"
+	"github.com/stripe/stripe-go/customer"
+	"gopkg.in/spacemonkeygo/monkit.v2"
+)
+
+var mon = monkit.Package()
+
+// Service is an implementation for PaymentsService via Stripe and Coinpayments
```

Comment:
this service will implement not only Accounts interface, but Transactions, Invoices, etc..

Reply:
Does `PaymentsService` currently exist?

Reply:
@ifraixedes currently now not. And i'm not sure that it will

Reply:
that's the point of my confusion, I thought that the comment was wrong because I couldn't find that type and for that reason, I had that suggestion. 

I don't mind if `payments/Account` is there or not, but having some unexisting identifier leads to confusion. Could you think to write something different?